e2e/Enclave.mk

```diff
@@ -52,7 +49,8 @@ endif
 
 SGX_COMMON_CFLAGS += -fstack-protector
 
-CARGO_FEATURES = --features=default
+ENCLAVE_CARGO_FEATURES = --features=default
+APP_CARGO_FEATURES     = --features=default
```

Comment:
Looks like `APP_CARGO_FEATURES` is unused and not necessary. I think these changes (*_CARGO_FEATURES) are unnecessary.